posthog/tasks/process_event.py

```diff
@@ -237,12 +238,14 @@ def process_event(
 
     properties = data.get("properties", data.get("$set", {}))
 
-    _capture(
-        ip=ip,
-        site_url=site_url,
-        team_id=team_id,
-        event=data["event"],
-        distinct_id=distinct_id,
-        properties=properties,
-        timestamp=handle_timestamp(data, now, sent_at),
-    )
+    # Selectively block certain teams from having events published to Postgres on Posthog Cloud
+    if not getattr(settings, "MULTI_TENANCY", False) or team_id not in [536, 572]:
```

Comment:
should maybe make this an env var or some sort of config or inject this during posthog-prod